File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/configs/BeanConfigs.java
package org.ilyatyamin.yacontesthelper.configs;

import com.fasterxml.jackson.databind.ObjectMapper;
import org.springframework.context.annotation.Bean;

public class BeanConfigs {
    @Bean
    ObjectMapper getObjectMapper() {
        return new ObjectMapper();
    }
}

File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/configs/BeanConfigs.kt
package org.ilyatyamin.yacontesthelper.configs

import feign.codec.ErrorDecoder
import org.springframework.context.annotation.Bean
import org.springframework.web.filter.CommonsRequestLoggingFilter


class BeanConfigs {
    @Bean
    fun errorDecoder(): ErrorDecoder {
        return CustomErrorDecoderFeign()
    }

    @Bean
    fun requestLoggingFilter(): CommonsRequestLoggingFilter {
        val loggingFilter = CommonsRequestLoggingFilter()
        loggingFilter.setIncludeClientInfo(true)
        loggingFilter.setIncludeQueryString(true)
        loggingFilter.setIncludePayload(true)
        loggingFilter.setMaxPayloadLength(64000)
        return loggingFilter
    }
}
